index.html

```diff
@@ -103,12 +103,73 @@
           <div class="description">Aloe Vera</div>
         </div>
       </div>
+    </div>
+    <div class="phone-number-container">
+      <div class="phone-number-container-headline">Naa, durstig geworden?</div>
+      <div class="description">Pick-up Bestellung unter</div>
```

Comment:
Description ist leider nicht wirklich aussagekräftig, wenn du es als generelle Klasse nutzt. Kannst je nachdem was du tust, in dem Klassennamen die Funktionen nennen. 
bspw. "light-grey-text". Dann weiß jede Person ungefähr worum es sich hier handelt, wenn die Klasse immer losgelöst von allem ist.